src/compose/ComposeText.js

```diff
@@ -85,6 +87,10 @@ class ComposeText extends React.Component {
   handleChangeText = (text: string) => {
     const { auth } = this.props;
     registerUserInputActivity(auth);
+    for (let i = 0; i < this.generalEmojiList.length; i++) {
```

Comment:
Should i make all emoji's related tasks in Emoji.js or emojiMap.js ? Accroding to me we can keep the map in emoji map and operation in Emoji.js .This way we do not have to keep track of the map we can abstract all the functionality in Emoji.js? What are your thoughts?